Release_Process.md

```diff
@@ -59,3 +59,12 @@ These steps are common between minor and patch releases:
         (without the 'v' in front) is _not_ active.
      3. Make sure the **stable** branch is _not_ active.
      4. Scroll to the bottom of the page and click the Submit button.
+1. Update all deployments with the currently released version accordingly.
```

Comment:
Shouldn't this is to be done post the release. That's when we update the deployments.
So, it would be a `release commit` for one, and a `update deployment` commit for the other.
Or should it all be in one commit and then just add docs to remind users to update their deployment in the end?